src/bundle/Resources/public/scss/_breadcrumb.scss

```diff
@@ -1,12 +1,22 @@
-.breadcrumb-item {
-    cursor: not-allowed;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: calculateRem(300px);
-    
-    + .breadcrumb-item::before,
-    &.active {
-        color: $ez-color-base-medium;
+.breadcrumb {
+    margin: calculateRem(24px) 0 calculateRem(16px);
+    flex-wrap: nowrap;
+
+    .breadcrumb-item {
+        cursor: not-allowed;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: calculateRem(300px);
+        
+        + .breadcrumb-item::before,
```

Comment:
Are you sure of these all selectors? This `before` looks strange, but if you're sure you can ignore this comment